### DISCUSSION TITLE
Is it working the status of the remote stations in Recent Sessions?

### DISCUSSION THREAD

**Discussion by mikekgr** [rustdesk/rustdesk#2786] [Q&A] [answered]
Dear Sirs,
I am running local Relay / Rendezvous Server and all remotes are latest 1.2.0 nightly build.
All of them, although they are online, in the status have orage lights on (instead of green as I think).
Do you know why? Can I make them to show correct online / offline status?
Many thanks and Best Regards,
Mike Kranidis
@rustdesk Please see this question and give me your answer. Thanks

![Recent Sessions](https://user-images.githubusercontent.com/15084551/211478680-dca390ba-488f-433c-8649-34001b777395.png)

**Comment by mikekgr** [comment]
@rustdesk Is thus status indication operational? Thanks

**Comment by mikekgr** [comment]
Hello somebody to answer???

**Comment by rustdesk** [comment, marked as answer]
try out latest self-hosted server
https://github.com/rustdesk/rustdesk-server/pull/76

**Reply by mikekgr** [reply]
Dear Sirs and Developers ( @rustdesk  ) of the marvelous RustDesk,
I have installed the version 1.1.6 long time ago, the installation that time, my server is Ubuntu 22.04.1 LTS) was from zip file not deb packages (as it is now). I want to upgrade to version 1.1.7 and I asking your help what can I do in order to keep all settings databases etc the same as it is now. The installation  is at /usr/local/bin/
and the related files are:
db_v2.sqlite3-shm
db_v2.sqlite3-wal
hbbr
hbbs
id_ed25519
id_ed25519.pub

Please help me.

Thanks and Best Regards,
Mike Kranidis

**Reply by rustdesk** [reply]
1.1.6 -> 1.1.7 has no database change.

**Reply by mikekgr** [reply]
@rustdesk 
so, in my case I only put the new executables hbbr  &   hbbs ? What about rustdesk-server-utils_1.1.7 that was not in 1.1.6 ?
Thanks a lot

**Reply by mikekgr** [reply]
Dear Sirs, Dear @rustdesk ,
I can conform that putting the new version (1.1.7) for hbbr , hbbs and restarting the server then I have correct online / offline reporting for the remotes
Thanks, I will mark this discussion as solved
